webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -976,6 +992,13 @@ const FlatRuns: React.FC<Props> = ({ projectId, searchId }) => {
               rowHeight={globalSettings.rowHeight}
               onRowHeightChange={onRowHeightChange}
             />
+            <FlatRunActionButton
+              isMobile={isMobile}
+              projectId={projectId}
+              selectedRuns={selectedRuns}
+              workspaceId={workspaceId}
+              onActionComplete={onActionComplete}
+            />
           </Row>
```

Comment:
add selected count?

Reply:
nvm I guess we created a separate ticket for this

Reply:
yep, there's a ticket for it.